src/App.js

```diff
@@ -93,13 +122,20 @@ function App() {
           </CustomizedDialogs>
         </BottomNavigation > 
       </Paper>  
+      {openDeleteModal.show && <DeleteModal 
+        titleNote={openDeleteModal.titleNote}
+        openDeleteModal={openDeleteModal} 
+        setOpenDeleteModal={setOpenDeleteModal} 
+        deleteNoteTrue={deleteNoteTrue}
+      />}
```

Comment:
Delete modal should work the same way we have other modal. Keep It clean and with nice naming.
To Delete modal you should pass and work with four props `open, note, onClose, onConfirm`